### PR TITLE
Jasmine

### PR DESCRIPTION
**Pull request by jasminecap**

### CODE REVIEW

File: file1.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts'This is jasmine's file'
```
Comment (abunsen) [inline file1.rb]: Not a fan of this style. Please use standard syntax. Like so: `Kernel.print("This is Jasmine's file")`

**Comment by hkdeven** [comment]
You can do it!

**Comment by abunsen** [comment]
:+1:

**Comment by FreeRick** [comment]
You did good

**Pull request merged**